src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -673,7 +673,7 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
             }
         }
 
-        dev->atomic_align = ucs_ilog2(arg_size);
+        dev->atomic_align = UCS_BIT(ucs_ilog2(arg_size));
```

Comment:
maybe can do `ucs_roundup_pow2` instead?

Reply:
need round down. arg_size is bit field of logs of possible sizes, here need max size.

Reply:
ok we have `ucs_align_down_pow2`)

Reply:
pls add ucs_round_down_pow2 to sys/math.h